source/DemoScene.cpp

```diff
@@ -80,5 +85,16 @@ bool DemoScene::update(Game &)
   if (Keyboard::getKeys()[OIS::KC_S]) {
     cameraNode->translate(0, 0, 10);
   }
+
+  if (Keyboard::getKeys()[OIS::KC_U]) {
+    illidan.stopAnimation();
+  }
+  if (Keyboard::getKeys()[OIS::KC_O]) {
+    illidan.setAnimation("Move", false);
+  }
+  if (Keyboard::getKeys()[OIS::KC_P]) {
+    illidan.setAnimation("Stand", false);
+  }
+  illidan.updateAnimations(fe.timeSinceLastFrame);
```

Comment:
This could be the reason of the bug I'm having to. Is it possible that he shortly plays the first of the next animation while standing?